torch/csrc/jit/codegen/cuda/ops/normalization.cpp

```diff
@@ -156,18 +156,14 @@ ForwardNormResult layer_norm(
   return layer_norm(x, norm_shape.size(), weight, bias, eps);
 }
 
-ForwardNormResult layer_norm(
-    TensorView* x,
+void layer_norm_axes(
```

Comment:
nit:
maybe rename to something more descriptive like `norm_axes_from_num_dims`

Reply:
or maybe `norm_properties_from_num_dims` since it's axes/mask/num_features
Maybe this would also be good to have in a resultsStruct instead of passing by ref